Makefile

```
@@ -127,7 +127,7 @@ $(DOCKER_DEMO_TARGETS):
 	docker build -t $(CTR_REGISTRY)/$(NAME):$(CTR_TAG) -f dockerfiles/Dockerfile.$(NAME) demo/bin/$(NAME)
 
 docker-build-init:
-	docker build -t $(CTR_REGISTRY)/init:$(CTR_TAG) -f dockerfiles/Dockerfile.init init
+	docker build -t $(CTR_REGISTRY)/init:$(CTR_TAG) -f dockerfiles/Dockerfile.init .
```

If no context is needed, you can do `docker build -` to signal that. That way no time is wasted copying the whole repo and any locally built binaries into the build context.
```suggestion
	docker build -t $(CTR_REGISTRY)/init:$(CTR_TAG) - < dockerfiles/Dockerfile.init
```
https://docs.docker.com/engine/reference/commandline/build/#build-with--

---

Didn't know, thanks!